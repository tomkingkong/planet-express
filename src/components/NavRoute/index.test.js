import React from 'react';
import { shallow, render } from 'enzyme';
import { MemoryRouter } from 'react-router';

import { NavRoute } from '.';

describe('NavRoute component', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<NavRoute />);
    expect(wrapper).toMatchSnapshot();
  });

  it('should match snapshot with /profile path', () => {
    const wrapper = shallow(<NavRoute location={{pathname: '/profile'}} />);
    render(
      <MemoryRouter initialEntries={['/profile']}>
        <NavRoute location={{pathname: '/profile'}} />
      </MemoryRouter>
    )
    expect(wrapper).toMatchSnapshot();
  });

  it('should match snapshot with / path', () => {
    const wrapper = shallow(<NavRoute location={{pathname: '/'}} />);
    render(
      <MemoryRouter initialEntries={['/']}>
        <NavRoute location={{pathname: '/'}} />
      </MemoryRouter>
    )
    expect(wrapper).toMatchSnapshot();
  });
});