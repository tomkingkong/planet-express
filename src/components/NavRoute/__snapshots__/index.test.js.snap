// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavRoute component should match snapshot 1`] = `
<div>
  <Route
    path="/"
    render={[Function]}
  />
  <Route
    component={[Function]}
    exact={true}
    path="/"
  />
  <Route
    component={[Function]}
    exact={true}
    path="/ISS"
  />
</div>
`;

exports[`NavRoute component should match snapshot with / path 1`] = `
<div>
  <Route
    path="/"
    render={[Function]}
  />
  <Route
    component={[Function]}
    exact={true}
    path="/"
  />
  <Route
    component={[Function]}
    exact={true}
    path="/ISS"
  />
</div>
`;

exports[`NavRoute component should match snapshot with /profile path 1`] = `
<div>
  <Route
    path="/"
    render={[Function]}
  />
  <Route
    component={[Function]}
    exact={true}
    path="/"
  />
  <Route
    component={[Function]}
    exact={true}
    path="/ISS"
  />
</div>
`;
